File: rust-sass-libsass-tests/ecosystem/smokes/python/smoke.py
import sass

assert sass.libsass_version == "3.6.6", sass.libsass_version
out = sass.compile(string="$color: red; .foo { color: $color; }")
assert out == ".foo {\n  color: red; }\n", repr(out)
print("python SMOKE-OK")

File: rust-sass-libsass-tests/ecosystem/smokes/lua/smoke.lua
local sass = require "sass"
assert(sass.LIBSASS_VERSION == "3.6.6", sass.LIBSASS_VERSION)
local out = assert(sass.compile("$color: red; .foo { color: $color; }"))
assert(out == ".foo {\n  color: red; }\n", string.format("%q", out))
print("lua SMOKE-OK")
